### PR TITLE
feat(feedback): add associated crash report to feedback details

### PR DESCRIPTION
**Pull request by michellewzhang**
Link the crash report issue, if it exists, in the feedback details.

placeholder while loading

https://github.com/getsentry/sentry/assets/56095982/43fe70ac-4065-4f63-a425-3132706d78a4

SCR-20231106-takm

Closes https://github.com/getsentry/team-replay/issues/274

### CODE REVIEW

File: static/app/components/feedback/feedbackItem/crashReportSection.tsx [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+}
+
+export default function CrashReportSection({
+  crashReportId,
+  organization,
+  projSlug,
+}: Props) {
+  const eventEndpoint = `/projects/${organization.slug}/${projSlug}/events/${crashReportId}/`;
+  const {data: crashReportData} = useApiQuery<Event>([eventEndpoint], {staleTime: 0});
+  // need to fetch the issue data since the event data is not complete enough for the component
```
Comment (billyvg) [inline static/app/components/feedback/feedbackItem/crashReportSection.tsx:26]: Do we have an endpoint that fetches an issue from an event id?

File: static/app/components/feedback/feedbackItem/crashReportSection.tsx [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  const groupId = crashReportData?.groupID;
+  const issueEndpoint = `/organizations/${organization.slug}/issues/${groupId}/`;
+  const {data: groupData} = useApiQuery<Group>([issueEndpoint], {staleTime: 0});
```
Comment (billyvg) [inline static/app/components/feedback/feedbackItem/crashReportSection.tsx:29]: Does this mean that we'll go doing at least 3 queries?

1. eventEndpoint
2. issueEndpoint where groupdId = undefined
3. issueEndpoint where groupId = defined

File: static/app/components/feedback/feedbackItem/crashReportSection.tsx (hunk 1 above)
Comment (michellewzhang) [reply static/app/components/feedback/feedbackItem/crashReportSection.tsx:26]: not to my knowledge, this is what's documented :( 
image

**Review by billyvg — APPROVED** [review]
nice!

**Pull request merged**